targets/csharp/source/Plugins/CloudScript/source/PlayFabFunctionContexts.cs

```diff
@@ -108,33 +106,30 @@ public class FunctionPlayerPlayStreamContext<TFunctionArgument>
         /// </summary>
         /// <param name="request">The request incoming to an Azure Function from the PlayFab server</param>
         /// <returns>A new populated <c>FunctionPlayerPlayStreamContext</c> instance</returns>
-        public static async Task<FunctionPlayerPlayStreamContext<TFunctionArgument>> Create(HttpRequestMessage request)
+        public static async Task<FunctionPlayerPlayStreamContext<TFunctionArgument>> Create(HttpRequest request)
         {
-            using (var content = request.Content)
+            string body = await request.ReadBodyAsStringAsync();
+            var contextInternal = PlayFabSimpleJson.DeserializeObject<FunctionPlayerPlayStreamContextInternal>(body);
```

Comment:
You should not be using PlayFabSimpleJson directly.
You should be fetching the serializer from PluginManager.
We JUST fixed a bunch of hard coded refs to PlayFabSimpleJson which were breaking the SDK when customers replaced the serializer.